doc/conf.py

```diff
@@ -105,16 +105,18 @@ def _warn_node(self, msg, node):
 
 # General information about the project.
 project = u'Luigi'
-copyright = u'2014, Erik Bernhardsson and Elias Freider'
+copyright = u'2015, Erik Bernhardsson and Elias Freider'
```

Comment:
FYI This may be a better way of doing the copyright, taken from another package I maintain:

```python
copyright = u'2014{0}, Lynn Root'.format(
    u'-{0}'.format(year) if year != 2014 else u""
)
```